app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("Visitor must have the vaccine")
+        if (visitor.get("vaccine").get("expiration_date")

```

Comment:
visitor.get("vaccine") returns None and than you try to do
None.get() it is not possible. YOu can use default for .get()